GEOSldas_App/ldas_setup

```diff
@@ -69,60 +69,69 @@ class LDASsetup:
         if 'exeinp' in cmdLineArgs:
             # sample sub-command
             # by construction, we can have
-            # either: {'exeinp': False, 'batinp': 'lasgh'}    <-- 'lasgh'???
-            # or:     {'exeinp': True, 'batinp': None}
-            if cmdLineArgs['exeinp']:
+            # either: {'exeinp': False, 'batinp': True }
+            # or:     {'exeinp': True,  'batinp': False}
+            if   cmdLineArgs['exeinp']:
                 _printExeInputKeys(rqdExeInpKeys)
-            elif cmdLineArgs['batinp'] :
+            elif cmdLineArgs['batinp']:
                 _printRmInputKeys(rqdRmInpKeys, optSlurmInpKeys)
             else:
-                raise Exception('not recognized option')
+                raise Exception('unrecognized option')
             sys.exit(0)
 
         # ------
         # ./ldsetup.py setup ...
         # ------
         # Instance variables
-        self.exeinpfile = cmdLineArgs['exeinpfile']
-        self.batinpfile = cmdLineArgs['batinpfile']
-        exphome_ = cmdLineArgs['exphome'].rstrip('/')
-        assert os.path.isdir(exphome_) # exphome should exist
-        self.exphome = os.path.abspath(exphome_)
-        self.nymdb = cmdLineArgs['nymdb']
-        self.nhmsb = cmdLineArgs['nhmsb']
-        self.verbose = cmdLineArgs['verbose']
-        self.runmodel = cmdLineArgs['runmodel']
+        self.exeinpfile         = cmdLineArgs['exeinpfile']
+        self.batinpfile         = cmdLineArgs['batinpfile']
+        exphome_                = cmdLineArgs['exphome'].rstrip('/')
+        assert os.path.isdir(exphome_)                                   # exphome should exist
+        self.exphome            = os.path.abspath(exphome_)
+        self.verbose            = cmdLineArgs['verbose']
+
+        # command line args for coupled land-atm DAS
+        self.nymdb              = cmdLineArgs['nymdb']
+        self.nhmsb              = cmdLineArgs['nhmsb']
+        self.agcm_res           = cmdLineArgs['agcm_res']
+        self.bcs_version        = cmdLineArgs['bcs_version']
+        self.ladas_cpl          = cmdLineArgs['ladas_cpl']
```

Comment:
@weiyuan-jiang , @saraqzhang : I'm a bit confused about the need for self.ladas_cpl and also self.ladas_coupling defined below.  I assume there's a way to use just one local variable, but I'm not sure I would get the order of operations right.  The idea is to have ladas_cpl from the command line take precedence over what may be defined in the exeinp file.   Which makes me wonder if we even need to allow for LADAS_COUPLING to be defined in the exeinp file.   It looks like more cleanup is needed here.

Reply:
@gmao-rreichle    that's right , we do not need LADAS_COUPLING in exeinp file.

Reply:
@gmao-rreichle    in current version:  (1)  ladas_cpl gets the command arg.  (2) self.rqdExeInp['LADAS_COUPLING'] = self.ladas_cpl  (3) self.ladas_coupling = int(self.rqdExeInp.get('LADAS_COUPLING',0))  (4) self.ladas_coupling  controls a few things down the process   and (5) LADAS_COUPLING the value written into LDAS.rc.

Reply:
@gmao-rreichle    we probably can directly use self.ladas_coupling = cmdLineArgs('ladas_cpl')  , eliminate self.ladas_cpl